lib/parse-mvn.ts

```diff
@@ -21,6 +22,14 @@ export function parseTree(text: string, withDev) {
   return { ok: true, data };
 }
 
+export function parseVersions(text: string): {javaVersion: string, mavenVersion: string} {
+  const data = text.split(os.EOL);
+  const mavenVersion = data[0];
+  const javaVersion = data.find((line) => line.startsWith('Java version:')) || '';
```

Comment:
is it more useful to parse the version number here? to avoid work downstream

Reply:
TBH, I'm not entirely sure what data we will want in the close future, so I'd rather keep whole string coming from `mvn`. If it would help, I can rename var to `javaVersionFromMvn` or something like that.